File: dynamicDebug/src/main/java/com/cloud/dynamicdebug/MockTransferFileImpl.java
package com.cloud.dynamicdebug;

import com.cloud.disk.api.file.FileInfo;
import com.cloud.disk.api.file.TransferFile;

import javax.inject.Inject;

public class MockTransferFileImpl implements TransferFile {
    @Inject
    public MockTransferFileImpl() {
    }

    @Override
    public FileInfo upload(String s) {
        return new FileInfo();
    }

    @Override
    public FileInfo download(String s) {
        return new FileInfo();
    }
}

File: dynamicDebug/src/main/java/com/cloud/dynamicdebug/MockTransferFileImpl.kt
package com.cloud.dynamicdebug

import com.cloud.disk.api.file.FileInfo
import com.cloud.disk.api.file.TransferFile
import javax.inject.Inject

class MockTransferFileImpl @Inject constructor() : TransferFile {
    override fun upload(s: String): FileInfo {
        return FileInfo()
    }

    override fun download(s: String): FileInfo {
        return FileInfo()
    }
}